gateway-protocol/src/main/proto/gateway.proto

```diff
@@ -919,19 +919,21 @@ service Gateway {
 
     Errors:
       NOT_FOUND:
-        - the process instance referenced by `processInstanceKey` is not active
+        - no process instance exists with the given key, or it is not active
+        - no process definition exists with the given target definition key
```

Comment:
:question: Should we mention tenants here? We do it in some other commands, so we would be aligned.

Reply:
@koevskinikola Yes, we should update multi tenancy behaviour here as well as in the documentation. But at the moment, it is not decided what do with multi tenancy. Let's wait for the result and I will update here accordingly.

Also, rejection cases for events (see #15405 and #15406) needs to be updated.